test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/helpers/LiabilityLinkHelperSpec.scala

```diff
@@ -32,82 +32,84 @@ class LiabilityLinkHelperSpec extends ControllerSpec {
   private val helper =
     LiabilityLinkHelper(appConfig = config)
 
-  "Backlink for liability weight page" should {
-
-    "when liability weight is more than minimum weight" in {
-      val headers = Headers().add(HeaderNames.xRequestId -> "req1")
-      val journeyRequest =
-        new JourneyRequest(
-          authenticatedRequest =
-            authRequest(headers, user = PptTestData.newUser("123")),
-          aRegistration(
-            withLiabilityDetails(LiabilityDetails(weight = Some(LiabilityWeight(Some(10001)))))
+  "The Liability Link Helper " should {
+    "provide back page to the liability weight page " when {
```

Comment:
If we're going to use this context for these tests which are providing different URLs then I think this needs to be something like: "provide a suitable back link".